src/tools/info/ucx_info.c

```diff
@@ -304,7 +304,8 @@ int main(int argc, char **argv)
 
     if (print_flags & UCS_CONFIG_PRINT_CONFIG) {
         ucs_config_parser_print_all_opts(stdout, UCS_DEFAULT_ENV_PREFIX,
-                                         print_flags, &ucs_config_global_list);
+                                         print_flags, &ucs_config_global_list,
+                                         (const char**)(argv + optind));
```

Comment:
1.
if we have additional arguments after "-c" or "-f", will it still work?
`ucx_info -c FRAG`
`ucx_info -c FLAG -f`
According to the help message the above should work.

2. why is the argument treated as a list, while the documentation help message does not say anything about it?